dev/freestanding/femr/docker-compose.dev.hydrant.yaml

```diff
@@ -0,0 +1,12 @@
+# docker-compose development override for hydrant service
+---
+version: "3.7"
+services:
+  hydrant:
+    command: sh -c "cd /opt/app && flask run --host 0.0.0.0 --port ${PORT:-8000}"
+    environment:
+      FLASK_ENV: development
+      FLASK_APP: hydrant.app:create_app()
```

Comment:
This should already be a default in the [Dockerfile](https://github.com/uwcirg/hydrant/blob/develop/Dockerfile#L8). 
```suggestion
```